sympy/integrals/tests/test_transforms.py

```diff
@@ -705,6 +707,14 @@ def test_laplace_transform():
         f(w), w, t, plane=0) == InverseLaplaceTransform(f(w), w, t, 0)
     assert LT(f(t)*g(t), t, s) == LaplaceTransform(f(t)*g(t), t, s)
 
+    # The following test confirms that issue #17098 is solved
+    F, _, _ = laplace_transform(diff(y(t), t, 2)
+                                + y(t)
+                                - 4*(Heaviside(t - 1)
+                                - Heaviside(t - 2)), t, s)
+    G = laplace_transform_replace_ic(F, y, Y, t, s, [1, 0])
```

Comment:
I wonder if it would be better to have a function `laplace_transform_ode` that combines these steps so that you only need to call one function. Maybe it should just be possible to provide these other arguments directly to `laplace_transform`. There might be situations where you want to calculate a `laplace_transform` in terms of an arbitrary function but it's not even about ODEs like:
```python
In [6]: laplace_transform(exp(-t)*f(t), t, s)
Out[6]: LaplaceTransform(f(t), t, s + 1)
```

Reply:
Considering that there might be different undefined functions in an expression, could it be an idea to give the laplace_transform an argument `replacements=[]` that could be called with a list of tuples containing function pairs and initial conditions?  Example:

```
r = [(f, F, [1, 0]), (h, T, [0,0])]
laplace_transform(expr, t, s, replacements=r)
```